openjdk11/plan.sh

```diff
@@ -0,0 +1,60 @@
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_name=openjdk11
+# NOTE: Retrieve download link and shasum from here: https://jdk.java.net/11/
+pkg_version=11.0.2
+pkg_source=https://download.java.net/java/GA/jdk11/9/GPL/openjdk-${pkg_version}_linux-x64_bin.tar.gz
+pkg_shasum=99be79935354f5c0df1ad293620ea36d13f48ec3ea870c838f20c504c9668b57
+pkg_filename=openjdk-${pkg_version}_linux-x64_bin.tar.gz
+pkg_dirname="jdk-${pkg_version}"
+pkg_license=("GPL-2.0-only")
+pkg_description=('Oracle Java Runtime Environment. This package is made available to you to allow you to run your applications as provided in and subject to the terms of the Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX, found at http://www.oracle.com/technetwork/java/javase/terms/license/index.html')
```

Comment:
Is this the correct pkg description?

Reply:
@irvingpop The pkg description will need to get updated.

Reply:
my bad, missed these notifications.  fixes incoming!